docs/mvc-tutorial-part-7-lifestyles.md

```diff
@@ -16,7 +16,7 @@ That's however as far as we can get for what we want from `ISession`. When it's
 
 ### The `PerWebRequest` lifestyle
 
-To change the `ISession` lifestyle to be per web request, we need to specify that in the registration. So we need to change it to the following:
+Please install the `Castle.AspNet.SystemWebFacility` NuGet first. To change the `ISession` lifestyle to be per web request, we need to specify that in the registration. So we need to change it to the following:
```

Comment:
I do believe this documentation is super dated. I did the minimum here, but after changing this I think there is a bit of work that needs to happen here to clear out the cruft.